src/main/java/org/folio/rest/util/HelperUtils.java

```diff
@@ -55,6 +55,8 @@ public class HelperUtils {
   private static final String ERROR_CAUSE = "cause";
   private static final String ERROR_MESSAGE = "errorMessage";
   private static final Pattern CQL_SORT_BY_PATTERN = Pattern.compile("(.*)(\\ssortBy\\s.*)", Pattern.CASE_INSENSITIVE);
+  private static final Pattern ERROR_PATTERN = Pattern.compile("(message).*(code).*(parameters)");
+  private static final Pattern ERRORS_PATTERN = Pattern.compile("(errors).*(message).*(code).*(parameters)");
```

Comment:
Can you please rename to ERRORS_LIST_PATTERN and corresponding java methods also, because it is hard distingwish difference between these 2 names